client-ui/src/app/components/header/header.component.ts

```diff
@@ -32,6 +34,8 @@ export class HeaderComponent {
     private store: Store<AppState>
   ) { }
 
+  appVersion: string = __APP_VERSION__
```

Comment:
type definition here is redundant + it should be readonly.

Reply:
+1 for `readonly`.
For type definition, technically it is optional, but usually type defs helps a lot to reading if the type isn't obvious from the context. Also we require type defs for public members, and `appVersion` is a public member.

Reply:
What can I say ...

`
readonly varName: string = ''
`
is redundant in Typescript. 

So you can start thinking to not require it :)

I will mark that conversion as resolved.

Reply:
Completely agree, for literals! But `__APP_VERSION__` isn't a literal, it is a global constant which type isn't obvious in the place of `appVersion`.